data/README.md

```diff
@@ -39,18 +39,27 @@ instance, they should be in the same format as the files below.
   Contains signature (node) activity levels (as absolute values) for each
   sample in the compendium derived from the eADAGE machine learning model.
   In this file, samples are rows, and signatures are columns. This file is
-  based on the
-  `all-pseudomonas-gene-normalized_HWActivity_perGene_with_net300_100models_1_100_k=300_seed=123_ClusterByweighted_avgweight_network_ADAGE.txt`
-  file, which is now in the `data/old/` folder and was generated by
+  based on:
+  ```
+  all-pseudomonas-gene-normalized_HWActivity_perGene_with_net300_100models_1_100_k=300_seed=123_ClusterByweighted_avgweight_network_ADAGE.txt
+  ```
+
+  which is now in the `data/old/` folder and was generated by
   [this script](https://bitbucket.org/greenelab/eadage/src/tip/node_interpretation/write_HWactivity.R).
 
+  **Note:** The signatures shown on the web (click
```

Comment:
Great docs! Do you think it would be better to say "... on the web interface" or "... on the web UI" to be more specific?

Reply:
Sure. I will change "web" into "web interface".